schema/collection.go

```diff
@@ -39,7 +39,7 @@ type DefaultCollection struct {
 	// Id is the dictionary encoded value for this collection.
 	Id uint32
 	// SchVer returns the schema version
-	SchVer int
+	SchVer int32
```

Comment:
nit: since the version is always unsigned - use uint32 instead?

Reply:
avoiding the translation from uint32 --> int32 because in the payload we are storing it as int32.